server.js

```diff
@@ -161,7 +161,7 @@ ${beautify(accessTokenResp.data, null, 4)}`);
 			);
 
 		let apiResp = await axios.get(
-			`https://${conf.hostname}:${conf.port}/api/am/${conf.restapi}/${conf.version}/apis?expand=${conf.expand}&limit=${conf.limit}&offset=${conf.offset}&query=${conf.query}`,
+			`https://${conf.hostname}:${conf.port}/api/am/${conf.restapi}/v1/apis?expand=${conf.expand}&limit=${conf.limit}&offset=${conf.offset}&query=${conf.query}`,
```

Comment:
Let's introduce a new config in the `deployment.toml` to configure the version number. Hence, we can reuse the same by configuring from the TOML configs.